Mathlib/Geometry/Manifold/IsManifold/Basic.lean

```diff
@@ -172,7 +172,7 @@ lemma ModelWithCorners.range_eq_target {𝕜 E H : Type*} [NontriviallyNormedFie
 def ModelWithCorners.of_target_univ (𝕜 : Type*) [NontriviallyNormedField 𝕜]
     {E : Type*} [NormedAddCommGroup E] [NormedSpace 𝕜 E] {H : Type*} [TopologicalSpace H]
     (φ : PartialEquiv H E) (hsource : φ.source = univ) (htarget : φ.target = univ)
-    (hcont : Continuous φ) (hcont_inv : Continuous φ.symm) : ModelWithCorners 𝕜 E H where
+    (hcont : Continuous φ) (hcont_inv : Continuous φ.invFun) : ModelWithCorners 𝕜 E H where
```

Comment:
The normal form is `φ.symm`, so I'm not really happy with this change. Would a lemma like the following solve the issue?
```
@[fun_prop] lemma foo {α β : Type*} [TopologicalSpace α] [TopologicalSpace β]
    (e : PartialEquiv α β) (he : Continuous e.symm) : Continuous e.invFun := he
```